### PR TITLE
Fix login form layout for mobile devices -fixes #40

### LINKED ISSUE #40: Login form doesn't scale down nicely on mobile

### PR DESCRIPTION
**Pull request by Himani3010**

### CODE REVIEW

File: sass/plugins/charitable/_charitable-forms.scss [hunk 1]
```diff
@@ -264,6 +264,15 @@
     .button-primary {
         width: auto;
     }
+
+
```
Comment (ericnicolaas) [inline sass/plugins/charitable/_charitable-forms.scss]: Extra line; just have one empty line between CSS rules :)

File: sass/plugins/charitable/_charitable-forms.scss [hunk 2]
```diff
@@ -264,6 +264,15 @@
     .button-primary {
         width: auto;
     }
+
+    
+    @media screen and (max-width:38em) {
```
Comment (ericnicolaas) [inline sass/plugins/charitable/_charitable-forms.scss]: I think `max-width` should be `37.5em` which is what we use everywhere else.

File: sass/plugins/charitable/_charitable-forms.scss [hunk 3]
```diff
@@ -264,6 +264,15 @@
     .button-primary {
         width: auto;
     }
+
+    
+    @media screen and (max-width:38em) { 
+    	.login-username, .login-password {
```
Comment (ericnicolaas) [inline sass/plugins/charitable/_charitable-forms.scss]: Again, just a small nitpicky thing, but if you could have a new line after the comma that would be a little easier to read in the SASS files. i.e.:

```
.login-username,
.login-password {
   ...
}
```

File: sass/plugins/charitable/_charitable-forms.scss [hunk 4]
```diff
@@ -264,6 +264,15 @@
...
+        float: none;
+        width: 100%;
+        @include padding( 0 );
```
Comment (ericnicolaas) [inline sass/plugins/charitable/_charitable-forms.scss]: We can just use `padding: 0;` here instead. The `padding()` method is not needed in this case.

File: sass/plugins/charitable/_charitable-forms.scss (hunk 2 above)
Comment (Himani3010) [reply sass/plugins/charitable/_charitable-forms.scss]: Thanks, have adjusted and will take a note about this!

File: sass/plugins/charitable/_charitable-forms.scss (hunk 3 above)
Comment (Himani3010) [reply sass/plugins/charitable/_charitable-forms.scss]: Noted and adjusted 👍

File: sass/plugins/charitable/_charitable-forms.scss (hunk 4 above)
Comment (Himani3010) [reply sass/plugins/charitable/_charitable-forms.scss]: Adjusted in the commit 52c2473

File: sass/plugins/charitable/_charitable-forms.scss [hunk 5]
```diff
@@ -264,6 +264,15 @@
...
     }
+    
+    @media screen and (max-width:37.5em) { 
+        .login-username, 
+        .login-password {
+        float: none;
+        width: 100%;
+        padding: 0;
+    }
+}
```
Comment (ericnicolaas) [inline sass/plugins/charitable/_charitable-forms.scss]: @Himani3010 Just a small issue with the indentation here; the CSS rules should be indented. i.e.:

```
@media screen and (max-width:37.5em) { 
        .login-username, 
        .login-password {
            float: none;
            width: 100%;
            padding: 0;
        }
}
```

Comment (Himani3010) [reply sass/plugins/charitable/_charitable-forms.scss]: @ericnicolaas Fixed, thanks for notifying me!

**Pull request merged**